draft-ietf-quic-multipath.md

```diff
@@ -189,6 +189,13 @@ used in 1-RTT packets and ACK_MP frames to distinguish packet number
 spaces for different paths. Both identifiers have the same value, which
 is the sequence number of the connection ID, when a new path is established.
 
+The initial path that is used during the handshake (and multipath negotiation)
+has the path ID 0 and therefore all 0-RTT packets are also tracked and
+processed with the path ID 0.
+For 1-RTT packets, the path ID is the sequence number of
+the Destination Connection ID present in the packet header, as defined
+in {{Section 5.1.1 of QUIC-TRANSPORT}}.
```

Comment:
I'd suggest that we only keep Line 192 to 194, it's useful for readers to understand the initial path identifier.
Line 195 to 197 is very similar with the definition of Path Identifier in Line 175 to 177.

Reply:
That's ok.